src/main/java/teammates/logic/core/CommentsLogic.java

```diff
@@ -37,6 +38,9 @@
  */
 public class CommentsLogic {
     
+    private static final String ANONYMOUS_GIVER = "Anonymous";
+    private static final String ANONYMOUS_RECEIVER = "Anonymous";
```

Comment:
Move this to `Const.java` too perhaps. Maybe you could also rename it as `DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT` and just have one constant for `Anonymous`